consensus/istanbul/backend/engine_test.go

```diff
@@ -654,6 +655,159 @@ func makeExpectedResult(indices []int, candidate []common.Address) []common.Addr
+		{ // test the validators are updated properly when minimum staking is changed in none mode
+			[]uint64{8000000, 7000000, 6000000, 5000000},
+			[]vote{
+				{"governance.governancemode", "none"}, // voted on epoch 1, applied from 6-8
+				{"reward.minimumstake", "5500000"},    // voted on epoch 2, applied from 9-11
+				{"reward.minimumstake", "6500000"},    // voted on epoch 3, applied from 12-14
+				{"reward.minimumstake", "7500000"},    // voted on epoch 4, applied from 15-17
+				{"reward.minimumstake", "8500000"},    // voted on epoch 5, applied from 18-20
+				{"reward.minimumstake", "7500000"},    // voted on epoch 6, applied from 21-23
+				{"reward.minimumstake", "6500000"},    // voted on epoch 7, applied from 24-26
+				{"reward.minimumstake", "5500000"},    // voted on epoch 8, applied from 27-29
+				{"reward.minimumstake", "4500000"},    // voted on epoch 9, applied from 30-32
+			},
+			[]expected{
+				{[]uint64{0, 1, 2, 3, 4, 5, 6, 7, 8}, []int{0, 1, 2, 3}, []int{}},
+				{[]uint64{9, 10, 11}, []int{0, 1, 2}, []int{3}},
+				{[]uint64{12, 13, 14}, []int{0, 1}, []int{2, 3}},
+				{[]uint64{15, 16, 17}, []int{0}, []int{1, 2, 3}},
+				{[]uint64{18, 19, 20}, []int{0, 1, 2, 3}, []int{}},
+				{[]uint64{21, 22, 23}, []int{0}, []int{1, 2, 3}},
+				{[]uint64{24, 25, 26}, []int{0, 1}, []int{2, 3}},
+				{[]uint64{27, 28, 29}, []int{0, 1, 2}, []int{3}},
+				{[]uint64{30, 31, 32}, []int{0, 1, 2, 3}, []int{}},
+			},
+		},
+		{ // test the validators (including governing node) are updated properly when minimum staking is changed in single mode
+			[]uint64{5000000, 6000000, 7000000, 8000000},
+			[]vote{
+				{"reward.minimumstake", "8500000"}, // voted on epoch 1, applied from 6-8
+				{"reward.minimumstake", "7500000"}, // voted on epoch 2, applied from 9-11
+				{"reward.minimumstake", "6500000"}, // voted on epoch 3, applied from 12-14
+				{"reward.minimumstake", "5500000"}, // voted on epoch 4, applied from 15-17
+				{"reward.minimumstake", "4500000"}, // voted on epoch 5, applied from 18-20
+				{"reward.minimumstake", "5500000"}, // voted on epoch 6, applied from 21-23
+				{"reward.minimumstake", "6500000"}, // voted on epoch 7, applied from 24-26
+				{"reward.minimumstake", "7500000"}, // voted on epoch 8, applied from 27-29
+				{"reward.minimumstake", "8500000"}, // voted on epoch 9, applied from 30-32
+			},
+			[]expected{
+				// 0 is governing node, so it is included in th validators all the time
+				{[]uint64{0, 1, 2, 3, 4, 5, 6, 7, 8}, []int{0, 1, 2, 3}, []int{}},
+				{[]uint64{9, 10, 11}, []int{0, 3}, []int{1, 2}},
+				{[]uint64{12, 13, 14}, []int{0, 2, 3}, []int{1}},
+				{[]uint64{15, 16, 17, 18, 19, 20, 21, 22, 23}, []int{0, 1, 2, 3}, []int{}},
+				{[]uint64{24, 25, 26}, []int{0, 2, 3}, []int{1}},
+				{[]uint64{27, 28, 29}, []int{0, 3}, []int{1, 2}},
+				{[]uint64{30, 31, 32}, []int{0, 1, 2, 3}, []int{}},
+			},
+		},
+		{
+			// test the validators are updated properly if governing node is changed
```

Comment:
```suggestion
		{ // test the validators are updated properly if governing node is changed
```

Reply:
I made the comments consistent with others. PTAL.